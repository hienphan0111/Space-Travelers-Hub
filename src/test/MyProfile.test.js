import TestRenderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect.js';
import { Provider } from 'react-redux';
import store from 'redux/store';
import MyProfile from 'routes/myProfile.route';
import axios from 'axios';

jest.mock('axios', () => ({
  post: jest.fn(),
  get: jest.fn()
}))

describe('Render the missions list in profile page', () => {
  it('Renders the missions list in profile page correctly', () => {
    const missionsList = TestRenderer.create(
        <Provider store={store}>
          <MyProfile />
        </Provider>,
    ).toJSON();
    expect(missionsList).toMatchSnapshot();
  });
});