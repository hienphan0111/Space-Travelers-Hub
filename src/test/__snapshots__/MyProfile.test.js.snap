// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the missions list in profile page Renders the missions list in profile page correctly 1`] = `
<div
  className="profile-container"
>
  <div
    className="mission-joined"
  >
    <h4>
      My Missions
    </h4>
    <div
      className="mis-list"
    />
  </div>
  <div
    className="rockets-booked"
  >
    <h4>
      My Rockets
    </h4>
    <div
      className="rockets-container"
    />
  </div>
</div>
`;
